clj-refactor.el

```diff
@@ -831,10 +831,10 @@ A new record is created to define this constructor."
           (and (file-exists-p file) file))
         (let ((file (expand-file-name "build.boot" project-dir)))
           (and (file-exists-p file) file))
-        (let ((file (expand-file-name "pom.xml" project-dir)))
-          (and (file-exists-p file) file))
         (let ((file (expand-file-name "deps.edn" project-dir)))
           (and (file-exists-p file) file))
+        (let ((file (expand-file-name "pom.xml" project-dir)))
```

Comment:
I'd move this to the end of the `or` to be on the safe side - seems to me every other project type should have precedence over `pom.xml`. I'm guessing you might need to update `cljr--project-dir` as well.